cubedash/generate.py

```diff
@@ -142,7 +142,8 @@ def _load_products(index: Index, product_names) -> List[DatasetType]:
 @click.option("--force-concurrently", is_flag=True, default=False)
 @click.option(
     "--init-database/--no-init-database",
```

Comment:
The old option is still there. Is it valid to have both `--init-database` and `--init` ?

Reply:
Yep -- you can include multiple aliases in click options (eg. '-r' and '--recursive').

I added the shorter version as it seems more suitable and memorable for a frequently used option.